go/Makefile

```diff
@@ -6,12 +6,14 @@ install-dev-tools-go1.18.10:
 	go install mvdan.cc/gofumpt@v0.4.0
 	go install github.com/segmentio/golines@v0.11.0
 	go install honnef.co/go/tools/cmd/staticcheck@v0.3.3
+	cargo install --force cbindgen
```

Comment:
why `--force`?

Reply:
It was recommended in the cbindgen README to make it update to the latest version automatically, but probably not what we actually want here. May be irrelevant now since we may use Go bindings instead.